specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -0,0 +1,3793 @@
+    },
+    "RoutingPreference": {
+      "properties": {
+        "routingChoice": {
+          "type": "string",
+          "description": "Routing Choice defines the kind of network routing opted by the user.",
+          "enum": [
+            "MicrosoftRouting",
+            "InternetRouting"
+          ],
+          "x-ms-enum": {
+            "name": "RoutingChoice",
+            "modelAsString": true
+          }
+        },
+        "publishMicrosoftEndpoints": {
+          "type": "boolean",
+          "description": "A boolean flag which indicates whether microsoft routing storage endpoints are to be published"
+        },
+        "publishInternetEndpoints": {
+          "type": "boolean",
+          "description": "A boolean flag which indicates whether internet routing storage endpoints are to be published"
+        }
+      },
+      "description": "Routing preference defines the type of network, either microsoft or internet routing to be used to deliver the user data, the default option is microsoft routing"
+    },
+    "BlobRestoreStatus": {
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The status of blob restore progress. Possible values are: - InProgress: Indicates that blob restore is ongoing. - Complete: Indicates that blob restore has been completed successfully. - Failed: Indicates that blob restore is failed.",
+          "enum": [
+            "InProgress",
+            "Complete",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "BlobRestoreProgressStatus",
+            "modelAsString": true
+          }
+        },
+        "failureReason": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Failure reason when blob restore is failed."
+        },
+        "restoreId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Id for tracking blob restore request."
+        },
+        "parameters": {
+          "readOnly": true,
+          "$ref": "#/definitions/BlobRestoreParameters",
+          "description": "Blob restore request parameters."
+        }
+      },
+      "description": "Blob restore status."
+    },
+    "DeletedAccountProperties": {
+      "properties": {
+        "key": {
+          "readOnly": true,
+          "type": "string",
+          "description": "key of the deleted account"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "name of the deleted account"
+        },
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "location of the deleted account"
+        },
+        "subscription": {
+          "readOnly": true,
+          "type": "string",
+          "description": "subscription of the deleted account"
+        },
+        "resourceGroupName": {
```

Comment:
please just give the full resource ID of the original storage account instead of separating it out into components.  Follows keyVault's existing pattern.